crates/nu-command/src/commands/nth.rs

```diff
@@ -48,6 +50,11 @@ impl WholeStreamCommand for Nth {
                 example: "echo [first second third] | nth 0 2",
                 result: Some(vec![Value::from("first"), Value::from("third")]),
             },
+            Example {
+                description: "Skip the first and third rows",
+                example: "echo [first second third] | nth --skip 0 2",
```

Comment:
Can you use an example here that shows what `--skip` is doing in isolation? Seems like we may want to think about either selecting rows or skipping rows (and selecting all the other rows), but not both?

Reply:
You mean just `nth --skip` by itself? It will throw an error since you need to supply a row number. The `--skip` flag is meant to invert the `nth` selection. Without a row number it would output the unchanged stream, thus being pointless.

Or do you mean `nth --skip 0` (using just one row number)?

Reply:
You know, I misread it to be that `--skip` was taking a parameter. Now that I can read it correctly, it makes sense.

Yeah, looks good.